flang-vm.yml

```diff
@@ -0,0 +1,43 @@
+---
+- hosts: GCCVM
+  tasks:
+  - name: Install epel-repository
```

Comment:
Should be: "Install epel-repository and centos-release-scl" or "Install repositories"